src/lib/features/project-permissions/projectPermissionChecker.ts

```diff
@@ -0,0 +1,17 @@
+import { IUnleashStores } from '../../types';
+import { IProjectPermissionStore } from './projectPermissionStoreType';
+import { IProjectPermissionChecker } from './projectPermissionCheckerType';
+
+export class ProjectPermissionChecker implements IProjectPermissionChecker {
```

Comment:
if we don't have any more code we can use the store directly. You can call the store as read model.